src/prisma/schema.prisma

```diff
@@ -2972,6 +2972,7 @@ model UserWeeklyStats {
   user          Scout    @relation(fields: [userId], references: [id], onDelete: Cascade)
   gemsCollected Int
   week          String
+  rank          Int      @default(0)
```

Comment:
So long as the plan isn't to use "0" as a non-value, I'm fine with making it required.

But I just want to note that the UserStats table does not have to be builder-specific. AllSeasonStats include stats for both builders and scouts, for instance. But we can easily just make it optional when we cross that bridge!

Reply:
I think we can go with a optional rank since that would indicate it hasn't been added yet.